code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -401,10 +404,17 @@
 		new /obj/item/stack/sheet/hot_ice(holder)
 
 	SET_REACTION_RESULTS(freon_burn_rate * (1 + oxygen_burn_ratio))
+	air.reaction_results["fire"] += freon_burn_rate * (1 + oxygen_burn_ratio)
```

Comment:
We use reaction_results for the actual result of the reaction now. I suggest having this be set by SET_REACTION_RESULTS and then dealing with the effects in hotspot code.

Reply:
crap, the cleanup reintroduced the fires. one moment.